modules/ossm-architecture.adoc

```diff
@@ -36,16 +36,16 @@ The *control plane* manages and configures the proxies that make up the data pla
 
 {ProductName} also bundles the following Istio add-ons as part of the product:
 
-* *Kiali* - Kiali is the management console for {ProductName}. It provides dashboards, observability, and robust configuration and validation capabilities. It shows the structure of your service mesh by inferring traffic topology and displays the health of your mesh. Kiali provides detailed metrics, powerful validation, access to Grafana, and strong integration for distributed tracing with Jaeger.
+* *Kiali* - Kiali is the management console for {ProductName}. It provides dashboards, observability, and robust configuration and validation capabilities. It shows the structure of your service mesh by inferring traffic topology and displays the health of your mesh. Kiali provides detailed metrics, powerful validation, access to Grafana, and strong integration with the {JaegerShortName}.
 
 * *Prometheus* - {ProductName} uses Prometheus to store telemetry information from services. Kiali depends on Prometheus to obtain metrics, health status, and mesh topology.
 
-* *Jaeger* - {ProductName} supports Jaeger for distributed tracing. Jaeger is an open source traceability server that centralizes and displays traces associated with a single request between multiple services. Using Jaeger you can monitor and troubleshoot your microservices-based distributed systems.
+* *Jaeger* - {ProductName} supports the {JaegerShortName}. Jaeger is an open source traceability server that centralizes and displays traces associated with a single request between multiple services. Using the {JaegerShortName} you can monitor and troubleshoot your microservices-based distributed systems.
 
-* *Elasticsearch* - ElasticSearch is an open source, distributed, JSON-based search and analytics engine. Jaeger uses ElasticSearch for distributed storage and indexing for logging and tracing data.
+* *Elasticsearch* - ElasticSearch is an open source, distributed, JSON-based search and analytics engine. The {JaegerShortName} uses ElasticSearch for distributed storage and indexing for logging and tracing data.
```

Comment:
> uses ElasticSearch for distributed storage and indexing for logging and tracing data.

seems weird (I am not a native speaker though :))

I would simplify it as `uses Elastcsearch as persistent storage.` :)